### PR TITLE
61410 Add VBS cassette and use in existing specs

### PR DESCRIPTION
**Pull request by kjsuarez**
## Summary
We're moving mocked specs over to VCR where appropriate

## Testing done
- Specs green

## What areas of the site does it impact?
Backend, specs

## Acceptance criteria

- [ ]  specs green

### CODE REVIEW

File: spec/support/vcr_cassettes/dmc/submit_to_vbs.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
+---
+http_interactions:
```
Comment (LindseySaari) [inline spec/support/vcr_cassettes/dmc/submit_to_vbs.yml:2]: Can you please add a CODEOWNERS entry for spec/support/vcr_cassettes/dmc/submit_to_vbs.yml for your team please? Our dangerbot action should have caught this, but in the meantime, please add this entry

File: config/settings/test.yml [hunk 2]
```diff
@@ -309,13 +309,13 @@ mcp:
...
     host: fake_url.com:9000
-    base_path: /base/path
+    base_path: /vbsapi
     service_name: VBS
     mock: true
   vbs_v2:
-    url: https://fake_url.com:9000
-    host: fake_url.com:9000
-    base_path: /base/path
+    url: https://internal-dsva-vagov-staging-fwdproxy-1821450725.us-gov-west-1.elb.amazonaws.com:4491
```
Comment (LindseySaari) [inline config/settings/test.yml:316]: Do these settings need to be added to the vets-api manifest values for live envs?

Comment (kjsuarez) [reply config/settings/test.yml:316]: No

File: spec/support/vcr_cassettes/dmc/submit_to_vbs.yml (hunk 1 above)
Comment (kjsuarez) [reply spec/support/vcr_cassettes/dmc/submit_to_vbs.yml:2]: Addressed

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -1601,6 +1601,7 @@ spec/support/vcr_cassettes/debts/get_letters.yml     @department-of-veterans-aff
 spec/support/vcr_cassettes/decision_review/200_contestable_issues.yml     @department-of-veterans-affairs/Benefits-Team-1   @department-of-veterans-affairs/backend-review-group @department-of-veterans-affairs/va-api-engineers
 spec/support/vcr_cassettes/dgi/claimant_info_loa3.yml     @department-of-veterans-affairs/my-education-benefits   @department-of-veterans-affairs/backend-review-group @department-of-veterans-affairs/va-api-engineers
 spec/support/vcr_cassettes/dmc/download_pdf.yml     @department-of-veterans-affairs/vsa-debt-resolution   @department-of-veterans-affairs/backend-review-group @department-of-veterans-affairs/va-api-engineers
+spec/support/vcr_cassettes/dmc/submit_to_vbs.yml     @department-of-veterans-affairs/vsa-debt-resolution
```
Comment (LindseySaari) [inline .github/CODEOWNERS:1604]: You'll also need to add `@department-of-veterans-affairs/backend-review-group`

**Review by ScottyRJames — APPROVED** [review]
LGTM

**Comment by kjsuarez** [comment]
> Hi @kjsuarez, it looks like some of the CI checks are not passing. Could you address those, and then I'll be able to give it another review on behalf of backend-review-group. Thank you!

@gia-lexa The complaint is for a line of CODEOWNERS that isn't touched in this PR. Could you please justify how it's relevant to this code change?

**Review by ScottyRJames — APPROVED** [review]

**Pull request merged**